hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/txns/submission/TransactionPrecheckTest.java

```diff
@@ -91,7 +91,7 @@ void setUp() {
 
     @Test
     void abortsOnInactivePlatform() {
-        given(currentPlatformStatus.get()).willReturn(PlatformStatus.MAINTENANCE);
+        given(currentPlatformStatus.get()).willReturn(PlatformStatus.CHECKING);
```

Comment:
Is this a new Status `CHECKING` ?

Reply:
Good question...there isn't a `MAINTENANCE` status any more, and for the purposes of this test, I just needed something other than `ACTIVE`, so didn't think too hard about it.

Let me research a bit more.